### ISSUE TITLE
Implement Bzlmod in Bazel

### ISSUE THREAD

**Issue by meteorcloudy** [bazelbuild/bazel#13316]
Implement the new dependency management system proposed in https://docs.google.com/document/d/1moQfNcEIttsk6vYanNKIy3ZuK53hQUFq1b1r0rmsYVg/edit?usp=sharing

**Comment by meteorcloudy** [comment]
We'll first build a MVP in the bzlmod branch.

**Comment by Psoffie009** [comment]
> Implement the new dependency management system proposed in https://docs.google.com/document/d/1moQfNcEIttsk6vYanNKIy3ZuK53hQUFq1b1r0rmsYVg/edit?usp=sharing

**Comment by cpsauer** [comment]
@Wyverald, for toolmakers/library maintainers that should be sure to adopt before bzlmod is enabled by default:
Is there a new issue we should follow to stay abreast of when bzlmod will fully launch?

**Comment by Wyverald** [comment]
Bzlmod will "fully" launch with the release of Bazel 6.0, at which point the flag --experimental_enable_bzlmod will be renamed to just --enable_bzlmod.

Bzlmod will be enabled by default in Bazel 7.0 at the earliest. That would mean deprecating WORKSPACE support, which we're still a rather long way from.

Both of these will be featured prominently in the release notes of the major releases, so hopefully it's unlikely for you to miss them :)

**Comment by cpsauer** [comment]
Definitely. Super good to know. Thank you! Great to know the forward roadmap--and thanks for all your great efforts.
